examples/Lasso_Regression.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
FWIW, personally, I find using https://github.com/fredrikekre/Literate.jl much nicer than directly committing notebooks to the repo.

Reply:
I will see if it is easy to use, thanks.